pkg/util/azureclient/applens/applens_client.go

```diff
@@ -0,0 +1,137 @@
+package applens
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+// AppLens Client created from CosmosDB Client
+// (https://github.com/Azure/azure-sdk-for-go/blob/3f7acd20691214ef2cb1f0132f82115f1df01a8c/sdk/data/azcosmos/cosmos_client.go)
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+)
+
+// AppLens client is used to interact with the Azure AppLens service.
+type Client struct {
+	endpoint string
+	pipeline runtime.Pipeline
+}
+
+// Endpoint used to create the client.
+func (c *Client) Endpoint() string {
+	return c.endpoint
+}
+
+// NewClient creates a new instance of AppLens client with Azure AD access token authentication. It uses the default pipeline configuration.
+// endpoint - The applens service endpoint to use.
+// cred - The credential used to authenticate with the applens service.
+// options - Optional AppLens client options.  Pass nil to accept default values.
+func NewClient(endpoint, scope string, cred azcore.TokenCredential, o *ClientOptions) (*Client, error) {
+	return &Client{endpoint: endpoint, pipeline: newPipeline([]policy.Policy{runtime.NewBearerTokenPolicy(cred, []string{scope}, nil), &appLensBearerTokenPolicy{}}, o)}, nil
```

Comment:
I don't think we need our own `appLensBearerTokenPolicy`. The built-in BearerTokenPolicy should do.